.github/workflows/test_pull_requests.yml

```diff
@@ -166,7 +166,7 @@ jobs:
 
     - name: Install Regression Tool
       if: always() && matrix.run_regressions && steps.branch_build.outcome != 'failure'  # always run this step as long as we actually built
-      run: pip install energyplus-regressions
+      run: pip install git+https://github.com/NREL/EnergyPlusRegressionTool.git@CleanupsJuly2024  #energyplus-regressions
```

Comment:
The regression tool has a PR with tons of cleanups, so I want to test that out here.  Once I confirm it's happy, I'll merge that PR over there, tag it to get a new release on PyPi, and revert this line change so that it pip installs from PyPi again.